### DISCUSSION TITLE
Reflect resources to a different cluster

### DISCUSSION THREAD

**Discussion by Krisztian-Molnar** [emberstack/kubernetes-reflector#449] [Ideas]
Dear maintainers of reflector,

This tool works like a charm within a cluster, it replicates our ElasticSearch secret containing certs to all of our namespaces where it's needed.
However it's only used on staging, because our production has a bit different setup: ES is deployed to another cluster, not where some of our apps are, and we use Internal Load Balancing to connect these apps from that different cluster to this ES.

Question: Would it be possible to somehow use ILB for replicating secrets to a different cluster, not only to different namespaces?

Thanks in advance!

**Comment by winromulus** [comment]
Right now no. This is not part of reflector's scope. But a good suggestion. I'm converting this to a discussion to get more feedback

**Reply by Krisztian-Molnar** [reply]
Thank you! 🙏

**Reply by emmeowzing** [reply]
This would be a really cool feature. Would love to work on it!
